.github/CODEOWNERS

```diff
@@ -5,6 +5,28 @@
 # taken by someone else.
 *                         @busunkim96
 
+##################################
+# Storage and Database (SoDa) team 
+##################################
+bigtable/                   @tseaver @crwilcox @frankyn
+docs/bigtable/              @tseaver @crwilcox @frankyn
+
+datastore/                  @tseaver @crwilcox @frankyn      
+docs/datastore              @tseaver @crwilcox @frankyn 
+
+storage/                    @tseaver @crwilcox @frankyn
+docs/storage                @tseaver @crwilcox @frankyn
+
+firestore/                  @tseaver @crwilcox @frankyn
+docs/firestore/             @tseaver @crwilcox @frankyn
+
+spanner/                    @tseaver @crwilcox @frankyn
+docs/spanner                @tseaver @crwilcox @frankyn
```

Comment:
Rather than perpetuating the churn in this file, would it make more sense to create teams in `sloth` and then update this file to use them?  E.g., `yoshi-soda`, `yoshi-animl`, `yoshi-stackdriver`?

Reply:
The yoshi-* ones seem to be maintained in Sloth, but there are others (like api-bigquery) that I assume were created manually. I'm not quite sure what the story is with teams, I'll bring this up in the team chat.

Thank you for the suggestion!